teachertool/src/transforms/removeCriteriaFromRubric.ts

```diff
@@ -14,6 +15,7 @@ export function removeCriteriaFromRubric(instance: CriteriaInstance) {
     };
 
     dispatch(Actions.setRubric(newRubric));
+    AutorunService.poke();
```

Comment:
This is fine, but if you wanted, I think you could also just put this in the reducer when the rubric is changed.

Reply:
You're also already calling it in setRubric so it's fairly redundant here. But if you want to be explicit, I don't think it's harming anything.

Reply:
I tried that, but was getting very bizarre startup errors from react. The app would load to a blank page, with an error message in the console stating something about `stateAndDispatch` being accessed before initialization. I couldn't catch it in the debugger or figure out exactly what was happening. AFAICT, stateAndDispatch was not being referenced at all at startup. autorun does reference it in its timeout callback, and that is where the error was stemming from, but the function wasn't executing at startup. Bewildered, I moved the calls out to where they are now.

Reply:
Fair enough...setRubric is probably still sufficient, but since you have the debounce anyway, I think it's fine regardless.

Reply:
This codepath doesn't call the setRubric transform, but maybe it should.

Reply:
Ahh. Yeah, my bad. I think originally, I didn't do that because I was doing validation in setRubric, but I moved that out and never updated this code. I can do it in a separate change.